# Description: Improve the upstream manpage.
#  The upstream manpage is missing a few things, and a few of the
#  keybindings are incorrect.  There are also a few Lintian warnings.
# Author: Kenneth J. Pronovici <pronovic@debian.org>
Index: sopwith-1.8.3/doc/sopwith.6
===================================================================
--- sopwith-1.8.3.orig/doc/sopwith.6
+++ sopwith-1.8.3/doc/sopwith.6
@@ -5,7 +5,7 @@ sopwith \- classic aerial combat shoot e
 
 .SH SYNOPSIS
 .B sopwith 
-[ \-n | \-s | \-c | \-l | \-j \fIhost\fR ] [-f] [-2] [-x] [-q]
+[ \-n | \-s | \-c | \-l | \-j \fIhost\fR ] [\-f] [\-g\fIlevel\fR] [\-2] [\-x] [\-q] [\-p] [\-e]
 
 .SH DESCRIPTION
 Sopwith is a classic 1980s shoot 'em up originally developed by BMB
@@ -16,8 +16,13 @@ include emulation of the original PC spe
 sound output, working networking and various extra features that were
 included by the author in the released sources.
 
+.SH AUDIO
+As of version 1.7.3, audio is disabled by default.  Audio can be enabled
+using the \-p switch.  Once you're in the game, you will also have to
+use the S key to turn on sound effects.
+
 .SH CONTROLS
-The keys for controling the player's plane are defined
+The keys for controlling the player's plane are defined
 in the file ~/.sopwith/keys.
 The standard controls are as follows:
 .TP 
@@ -45,14 +50,20 @@ drop bomb
 H
 autopilot plane to home base
 .TP
-C
-launch flare
-.TP
 V
-fire missile
+launch starburst/flare (if missiles are enabled with \-x)
+.TP
+M
+fire missile (if missiles are enabled with \-x)
+.TP
+S
+turn on sound effects (if sound is enabled with \-p)
 .TP
 Ctrl+C
 quit
+.TP
+Ctrl+C Ctrl+C Ctrl+C
+quit immediately
 
 .SH OPTIONS
 .TP
@@ -75,8 +86,12 @@ by \fIhost\fR
 \-f
 Start in full screen mode if possible.
 .TP
+\-g\fIlevel\fR
+Start the game on the indicated difficulty level.  For instance, use \-g2 to start
+the game on level 2.  The default is level 0.
+.TP
 \-2
-Double-size the display window
+Double\-size the display window
 .TP
 \-x
 Enable missiles
